### DISCUSSION TITLE
Lanciare una routine Alexa da un pulsante fisico

### DISCUSSION THREAD

**Discussion by r1si** [indomus/forum#2811] [HUB: Altro (Homey, iHost, SmartThings, Fibaro ecc.)]
Buongiorno, 
nel lungo processo di domotizzazione della mia casa, al momento ho trovato un equilibrio sfruttando le routine di Alexa. 
Per il momento non ho ancora avuto la necessità di usare HA...

Attualmente il mio obiettivo è quello di posizionare dei pulsanti wifi che possano richiamare le routine di alexa, nei punti strategici.

Per esempio la mia attuale routine Alexa "Esco di casa" prevede:

- Chiama la routine "Esco di casa" di BTicino -> (Abbassa tutte le tapparelle e spegni le luci)
- Dopo 30 secondi Arma tutte le telecamere di video sorveglianza di blink
- Dopo 30 secondi Attiva l'allarme Ring
- Dopo 30 secondi chiude la serratura di casa

Inizialmente volevo comprare Sonoff R5, ma leggendo qua a e la non riesco a capire se verrà mai visualizzato dei dispositivi casa intelligente di alexa...
Leggevo anche che una soluzione proposta da molti è creare dei pulsanti virtuali (che a loro volta funzionino con i pulsanti veri) all'interno di smartthings.
Qualcuno conosce una soluzione al mio problema?

Grazie mille

**Comment by KC-inDomus** [comment]
> Per il momento non ho ancora avuto la necessità di usare HA...

è proprio questo il caso...
